client/wfnews-war/src/main/angular/src/app/components/report-of-fire/comment-page/rof-comments-page.component.html

```diff
@@ -5,10 +5,10 @@
     <mat-label>Note</mat-label>
   </div>
   <mat-form-field  class="text-container">
-    <textarea matInput #commentText rows="3" style="height: calc(100vh - 400px);"
```

Comment:
Side note, but did we verify that the maxLength in the database is actually 500?